src/frontend/app/Components/AddEditView.tsx

```diff
@@ -138,7 +138,7 @@ const AddEditView: React.FunctionComponent<AddEditViewProps> = (props: AddEditVi
   const onFormSubmit = (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
     if (modalVariant === 'create' || modalVariant === 'edit') {
-      saveFilter();
+      saveFilter().then();
```

Comment:
What are we trying to do with the `.then()` here?  Looking more broadly at this file we don't see have to any error handling for the form submission.  Is that happening under the hood somehow?